### ISSUE TITLE
Barebones Chirp CSV Export

### ISSUE THREAD

**Issue by tarxvftech** [M17-Project/REPEATER#3]
Make a view that allows for creating and downloading CSV files in the [Chirp](https://chirp.danplanet.com/projects/chirp/wiki/CSV_HowTo) format, like so:
```
Location,Name,Frequency,Duplex,Offset,Tone,rToneFreq,cToneFreq,DtcsCode,DtcsPolarity,Mode,TStep,Skip,Comment,URCALL,RPT1CALL,RPT2CALL,DVCODE
1,6m Call,52.525000,,0.000000,,88.5,88.5,023,NN,FM,5.00,,,,,,
2,2m Call,146.520000,,0.000000,,88.5,88.5,023,NN,FM,5.00,,,,,,
3,220 Call,223.500000,,0.000000,,88.5,88.5,023,NN,FM,5.00,,,,,,
4,70cm Call,446.000000,,0.000000,,88.5,88.5,023,NN,FM,5.00,,,,,,
```
The Location column is just a row number, and should start at 1 for the first data row and increase monotonically for each row.
Name and Frequency are the only fields you need to fill in, as the rest are undocumented or finicky at the moment and can be saved for later work. You can see many fields in the example above are empty, as there's nothing between the commas that separate fields.
Use a Repeater model's `freq.out_freq` value for the Frequency column.

The view should just list all repeaters, for now.

**Comment by Kabiirk** [comment]
Hi,

I'm new to django, but as per my understanding views are functions in django. and you want to create a view(function) in ```views.py``` that allows you to export data as CSV (Chirp format) from the database shown at https://repeater.m17project.org/repeater (screenshot below for reference)
![image](https://user-images.githubusercontent.com/16386226/195680276-091f08d0-cc70-4629-827b-3b47be28874f.png)

Please correct wherever required. If my understanding is correct, please assign this issue to me.

Regards

**Comment by tarxvftech** [comment]
Yes! :grin:  Please also attach the view in urls.py so it can be used (perhaps at `/repeaters/csv`? Your choice)
My goal is to have a button on that page you have screenshotted that will export the database in this Chirp format. 

Also, if it's possible, I'd like visiting that view to download the csv file rather than showing text on the screen, but this is not necessary to be accepted. It _should_ however render the csv file directly, i.e. don't render it into a template.


Later (if you want another PR) I would like to add filtering based on name and tags in the database, so that the export only contains matching entries. That would involve some UI work too.

**Comment by tarxvftech** [comment]
Live on the site!
I'm going to keep this open until I confirm the file generated works in Chirp itself (which will probably take some small tweaks to the fields, e.g. newlines and the way Chirp handles frequencies) but it's largely done thanks to @Kabiirk. Thank you! :grin:

**Comment by tarxvftech** [comment]
You can ignore the github CI error, it did actually deploy just fine. I'm looking into why it's reporting 'failing' even though it worked fine.
Update: Looks like flyctl somehow forgot how to find the app? even though it deploy fined. I deployed from my laptop and that seems to have reset it. Other people have reported this issue in the fly forums and github issues without a solid response, so i expect it'll pop up again.

**Comment by Kabiirk** [comment]
You're welcome, thanks for your guidance !
